src/frontend/api.js

```diff
@@ -6,13 +6,18 @@ import nodejs from "nodejs-mobile-react-native";
 import RNFS from "react-native-fs";
 import debug from "debug";
 import flatten from "flat";
+import DeviceInfo from "react-native-device-info";
+import AppInfo from "./lib/AppInfo";
 
 import type {
   Preset,
   Field,
   Metadata,
   Messages,
 } from "./context/ConfigContext";
+
+import { version as APP_VERSION } from "../../package.json";
```

Comment:
The actual version of Mapeo Mobile (returned by `react-native-device-info`) includes a build number, e.g. `2.4.1.839`. There are scenarios where we might have a broken build and release a new build that only increments the build number. Also, we _may_ change the build process where the version in `package.json` is not kept in sync, so I think better to read the version from `DeviceInfo` which will always be set.